apps/client-server/src/app/account/account.service.ts

```diff
@@ -274,6 +275,17 @@ export class AccountService
     await instance.setWebsiteData(setWebsiteDataRequestDto.data);
   }
 
+  async customRoute(customRouteDto: CustomWebsiteRouteDto) {
+    this.logger.info(`Custom website route for '${customRouteDto.id}'`);
+
+    const account = await this.findById(customRouteDto.id, {
+      failOnMissing: true,
+    });
+    const instance = this.websiteRegistry.findInstance(account);
+
+    return instance.onCustomRoute[customRouteDto.route](customRouteDto.data);
```

Comment:
wasn't sure how to add custom routes, but atleast this approach is strictly typed